AutomatedTesting/Gem/PythonTests/AWS/Windows/client_auth/aws_client_auth_automation_test.py

```diff
@@ -3,46 +3,57 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
+
+
+Prerequisites for running AWS automation tests on your local Windows machine:
+1) Go to the AWS IAM console and create an IAM role called o3de-automation-tests which adds your own AWS account as
```

Comment:
Minor: See changes above as we should the correct policy name which is AdministratorAccess